templates/default/package.json

```diff
@@ -45,7 +45,7 @@
     "rehype-mathjax": "^4.0.2",
     "rehype-prism-plus": "^1.4.2",
     "rehype-slug": "^5.0.1",
-    "remark-callouts": "^1.0.1",
+    "remark-callouts": "^1.0.2",
```

Comment:
As a very minor comment i think `remark-callouts` should have bumped at least a minor version e.g. `1.1.0` for what is a substantive change / fix.